### PR TITLE
Add initial AppVeyor configuration

### PR DESCRIPTION
**Pull request by chalcolith**
Adds an initial AppVeyor configuration.  Builds and zips, but does not deploy to BinTray or elsewhere.

### CODE REVIEW

**Review by jemc — APPROVED** [review]
LGTM - Feel free to merge at will after you get CI to pass.

**Comment by chalcolith** [comment]
Now BinTray is down.  The AppVeyor script downloads ponyc from Bintray, so i'll wait a while and try again.

BTW, I don't seem to have a commit bit in this repo.

**Comment by jemc** [comment]
Okay, thanks for letting me know about the commit bit.

The perms have been fixed to match the `ponyc` repo now.

**Pull request merged**
